DirectProgramming/C++SYCL_FPGA/ReferenceDesigns/cholesky/README.md

```diff
@@ -101,7 +98,6 @@ With this optimization, our FPGA implementation requires _n_ DSPs to compute the
 |:---                         |:---
 |`-Xshardware`                | Target FPGA hardware (as opposed to FPGA emulator)
 |`-Xsclock=<target fmax>MHz`  | The FPGA backend attempts to achieve <target fmax> MHz
-|`-Xsfp-relaxed`              | Allows the FPGA backend to re-order floating point arithmetic operations (for example, permit assuming $(a + b + c) == (c + a + b)$ )
```

Comment:
You removed the explanation of fp-relaxed but you kept this flag in the Windows flags in the `CMakeLists.txt`. Should you instead explain that this flag is necessary on Windows only (and why)?

Reply:
Updated. Remove fp-relaxed for all platform, added fp-precise for window's emulator only.